### PR TITLE
Updates Assignment Command and Assignment class

### LINKED ISSUE #66: Remove constructor from Assignment Class to neccesitate Student association at instantiation

### LINKED ISSUE #67: Update UI to show assignments to students

### PR DESCRIPTION
**Pull request by emmannyyy**
fixes #66  : This PR has updated the Assignment class to ensure that the constructor of the Assignment class will require a Student object for creation

fixes #67  : This PR updates the UI to ensure that the Assignments are shown when the `view_student` command is called. CSS has also been added

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddAssignmentCommand.java [hunk 1]
```diff
@@ -18,30 +26,42 @@ public class AddAssignmentCommand extends Command {
...
+            throw new CommandException(Messages.MESSAGE_INVALID_STUDENT_DISPLAYED_INDEX);
+        }
+        Student studentToAddAssignmentTo = lastShownList.get(index.getZeroBased());
+        Assignment toAdd = createAssignment(studentToAddAssignmentTo, toAddDescriptor);
+        if (studentToAddAssignmentTo.hasAssignment(toAdd)) {
             throw new CommandException(MESSAGE_DUPLICATE_ASSIGNMENT);
         }
-        model.addAssignment(toAdd);
+        Student edittedStudent = studentToAddAssignmentTo.addAssignment(toAdd);
+        model.setStudent(studentToAddAssignmentTo, edittedStudent);
```
Comment (lm-44) [inline src/main/java/seedu/address/logic/commands/AddAssignmentCommand.java:64]: Should rename "edittedStudent" to "editedStudent". Mispelling of "edited".

**Review by lm-44 — COMMENTED** [review]
Just a misspelling, other than that LGTM!

File: src/main/java/seedu/address/logic/commands/AddAssignmentCommand.java (hunk 1 above)
Comment (emmannyyy) [reply src/main/java/seedu/address/logic/commands/AddAssignmentCommand.java:64]: okay, fixed!

**Review by althea28 — APPROVED** [review]
looks good to me !

**Pull request merged**